test/JDBC/input/col_length-vu-prepare.sql

```diff
@@ -0,0 +1,217 @@
+CREATE DATABASE babel_3489_test_db;
+GO
+
+USE babel_3489_test_db;
+GO
+
+CREATE TABLE sys_column_length_test_table(
+    ID INT PRIMARY KEY,
+    col_char CHAR(10),
+    col_varchar VARCHAR(20),
+    col_binary BINARY(5),
+    col_varbinary VARBINARY(15),
+    col_nchar NCHAR(8),
+    col_nvarchar NVARCHAR(16),
+    col_text TEXT,
+    col_image IMAGE,
+    col_ntext NTEXT,
+    col_sysname sysname,
+    col_sql_variant SQL_VARIANT,
+    col_xml XML,
+    col_varcharmax VARCHAR(MAX),
+    col_nvarcharmax NVARCHAR(MAX),
+    col_varbinarymax VARBINARY(MAX),
+    col_bit BIT,
+    col_tinyint TINYINT,
+    col_bigint BIGINT,
+    col_smallint SMALLINT,
+    col_smallmoney SMALLMONEY,
+    col_money MONEY,
+    col_smalldatetime SMALLDATETIME,
+    col_real REAL,
+    col_float FLOAT,
+    col_time TIME,
+    col_datetime DATETIME,
+    col_datetime2 DATETIME2,
+    col_datetimeoffset DATETIMEOFFSET,
+    col_uniqueidentifier UNIQUEIDENTIFIER
+);
+GO
```

Comment:
Can you please create a table in another schema and write few tests around that?